pkg/builder/builder.go

```diff
@@ -0,0 +1,26 @@
+package builder
+
+// Builderer ...
+type Builderer interface {
```

Comment:
```suggestion
type Builder interface {
```

И дальше по коду также, зачем дополнительное 'er'